src/components/WebPlayBack.tsx

```diff
@@ -64,20 +66,56 @@ function WebPlayback(props: any) {
                 }
                 updatePlayerInfo(updatedPlayerInfo);
             });
-            
             player.connect();
-
             setPlayer(player);
         };
     }, []);
 
+    //method for updating the player objects.
     const updatePlayerInfo = (playerInfo: IPlayerUIProps) => {
         setPlayerInfo(playerInfo);
     }
+    //Method for playing and pausing music based on player state.
+    const toggle = (event: React.MouseEvent<HTMLElement>) => {
+        console.log("toggle clicked")
+        //@ts-ignore
+        statePlayer.getCurrentState().then(state => {
+            if (!state) {
+                //@ts-ignore
+                statePlayer.togglePlay().then(() => {
+                    console.log('player toggled: ' + state);
+                });
+            
+                return;
+            }
+            //@ts-ignore
+            statePlayer.togglePlay().then(() => {
```

Comment:
I don't think we need to get the current state to call togglePlay. We can probably simplify this by just calling `statePlayer.togglePlay` immediately without getting the current state first. This should probably look more like the `forward` and `backward` functions below

Reply:
This has been fixed thanks for the hints.